modules/ap.sh

```diff
@@ -4,6 +4,7 @@ function ap {
   mode=$1
   essid=$2
   password=$3
+  base_ip=$4
```

Comment:
ip = quadruple of 0..255 (2 digit hex number) delimited by . (dot) 

it seems you ip is just a triple ...

in other words we should accept quadruples and then remove last number and dot